<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoViewClass</class>
 <widget class="QMainWindow" name="StereoViewClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StereoView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="CameraState">
      <property name="title">
       <string>相机状态</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="RefreshCameraState">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>手动刷新相机状态</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_ChangeCamera">
             <property name="text">
              <string>左右相机交换</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>左相机</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LeftCameraState">
             <property name="styleSheet">
              <string notr="true">background-color:transparent</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>右相机</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="RightCameraState">
             <property name="styleSheet">
              <string notr="true">background-color:transparent</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="StereoCalibration">
      <property name="title">
       <string>相机矫正</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>瞳距调节</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="m_PdSlider">
             <property name="minimum">
              <number>-50</number>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_Pdedit">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="m_LockedCalibration">
           <property name="text">
            <string>应用矫正</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QCheckBox" name="m_useCalibrate">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>启用相机矫正（仅在相机畸变或角度差较大时使用）</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="m_useyml">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>使用标定参数矫正</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLineEdit" name="m_Calibrationfileroot">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_Calibrateroot">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="m_Manual">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>手动调整</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>左相机角度</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="m_LeftangleSlider">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <number>-300</number>
                 </property>
                 <property name="maximum">
                  <number>300</number>
                 </property>
                 <property name="pageStep">
                  <number>10</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="sliderPosition">
                  <number>0</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="m_Leftangleedit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>右相机角度</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="m_RightangleSlider">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <number>-300</number>
                 </property>
                 <property name="maximum">
                  <number>300</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="m_Rightangleedit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>光轴角度</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="m_MainangleSlider">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <number>-200</number>
                 </property>
                 <property name="maximum">
                  <number>200</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="sliderPosition">
                  <number>0</number>
                 </property>
                 <property name="tracking">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="m_Mainangleedit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>画面保留</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_remainall">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>全画面保留</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="m_remainmax">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>最大画面保留</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="Camerainitialization">
      <property name="title">
       <string>初始化参数（左右相机一致，非特殊相机不允许更改相机参数）</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_Width"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>对比度</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="2">
          <widget class="QLineEdit" name="m_Hue"/>
         </item>
         <item row="3" column="4">
          <widget class="QLineEdit" name="m_Exposure"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>饱和度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>宽度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>高度</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_Height"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="m_Brightness"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>帧数</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="2">
          <widget class="QLineEdit" name="m_Saturation"/>
         </item>
         <item row="0" column="3" colspan="2">
          <widget class="QLineEdit" name="m_Contrast"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>亮度</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="m_Fps"/>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>曝光度</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>色调</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="m_GetLeftCameraParameter">
             <property name="text">
              <string>获取左相机参数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_GetRightCameraParameter">
             <property name="text">
              <string>获取右相机参数</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="m_SetLeftCameraParameter">
             <property name="text">
              <string>设置左相机参数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_SetRightCameraParameter">
             <property name="text">
              <string>设置右相机参数</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>视屏控制</string>
      </property>
      <widget class="QLabel" name="m_CudaState">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>29</y>
         <width>120</width>
         <height>17</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>29</y>
         <width>961</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="m_UseCuda">
            <property name="text">
             <string>使用CUDA</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_Unfullscreen">
            <property name="text">
             <string>取消全屏</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="m_Startbutton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_Resetbutton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
